docs/1-install-and-setup.md

```diff
@@ -155,22 +155,11 @@ And that's it! If it's totally *borked*, file an issue and let us know.
 
 #### 1. Cannot find namespace 'firebase'.
 
-If you run into this error while trying to invoke `ng serve`, open `src/tsconfig.json` and add the "types" array as follows:
-
-```json
-{
-  "compilerOptions": {
-    ...
-    "typeRoots": [
-      "../node_modules/@types"
-    ],
-    
-    // ADD THIS
-    "types": [
-      "firebase"
-    ]
-  }
-}
+If you run into this error while trying to invoke `ng serve`, you need to update to the correct types with the following commands (from the root of a new project):
+
```

Comment:
@taylorjason2016 I can confirm now, that the PR you mentioned is not required. Just completed a fresh install on Ubuntu 16.0.4 LTS (Amazon Instance) and everything worked as expected, as mentioned in the install guide.

Note:- There seems to be a bug with installation of angular-cli on Ubuntu_16_0_4 itself, but that will be out of scope of angularFire2. I'll dig a little more, before reporting it. Hope this helps.